client/components/domains/connect-domain-step/connect-domain-step-advanced-start.jsx

```diff
@@ -6,21 +6,34 @@ import CardHeading from 'calypso/components/card-heading';
 import MaterialIcon from 'calypso/components/material-icon';
 import Notice from 'calypso/components/notice';
 import NoticeAction from 'calypso/components/notice/notice-action';
+import { isSubdomain } from 'calypso/lib/domains';
 import ConnectDomainStepWrapper from './connect-domain-step-wrapper';
 import { modeType, stepSlug, stepsHeading } from './constants';
 
 import './style.scss';
 
 export default function ConnectDomainStepAdvancedStart( {
 	className,
+	domain,
 	pageSlug,
 	mode,
 	onNextStep,
 	progressStepList,
 	setPage,
 } ) {
 	const { __ } = useI18n();
-	const switchToSuggestedSetup = () => setPage( stepSlug.SUGGESTED_START );
+	const firstStep = isSubdomain( domain )
+		? stepSlug.SUBDOMAIN_SUGGESTED_START
+		: stepSlug.SUGGESTED_START;
+	const switchToSuggestedSetup = () => setPage( firstStep );
+
+	const message = isSubdomain( domain )
+		? __(
+				'This is the advanced way to connect your subdomain, using CNAME records. We advise using our <a>suggested setup</a> instead, with NS records.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'This is the advanced way to connect your domain, using root A records & CNAME records. We advise using our <a>suggested setup</a> instead, with WordPress.com name servers.' )` *ES Score: 8*